packages/n/ncurses/xmake.lua

```diff
@@ -20,6 +20,9 @@ package("ncurses")
     end
 
     on_load(function (package)
+        if package:is_cross() then
+            package:add("deps", "ncurses~host", {kind = "binary", private = true})
```

Comment:
we need not add `~host` label, and modify kind, you can see https://github.com/xmake-io/xmake-repo/blob/1002ec9bf6d4be1bde579e073f73f61f1d61ccd4/packages/l/linux-headers/xmake.lua#L26